sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/GroupByKeyTest.java

```diff
@@ -186,6 +176,112 @@ public void testCombiningAccumulatingProcessingTime() throws Exception {
       p.run();
     }
 
+    /**
+     * Tests that data from a processing time trigger flows through subsequent GroupByKey
+     * transforms. To test this with TestStream, we check that it arrives in an early pane,
+     * demonstrating that the watermark did not cause the output. No runner appears to support
+     * {@link TestStream} processing time correctly for this, so we do it with actually processing
+     * time delays, which must be kept small so the test suite does not take forever. There are also
+     * bugs in runners around {@code Window.configure()} behaving differently than {@code
```

Comment:
should there be a JIRA for those bugs and referenced here?

Reply:
When I returned to this version of the test after writing the other version, I could not reproduce either issue. Using `Window.configure()` works, as does the processing time TestStream.